### DISCUSSION TITLE
How to indent headings?

### DISCUSSION THREAD

**Discussion by LukeDRussell** [minad/org-modern#180] [Q&A] [answered]
Hello,
I would like to have headings retain their indentation. So a second level heading has a space then the pretty bullet, third level heading has two spaces then bullet, etc. 
Is this possible?

**Comment by minad** [comment, marked as answer]
You can customize org-modern-hide-stars.

**Comment by LukeDRussell** [comment]
Got it. Note I returned to default `org-indent-indentation-per-level`

```elisp
(use-package org
 :hook (org-mode . visual-line-mode)
 :custom
 ...
 (org-startup-indented t)
 ;; (org-indent-indentation-per-level 1)
 (org-hide-emphasis-markers t)
 (org-pretty-entities t)
 ...
)

(use-package org-modern
  :hook (org-mode . global-org-modern-mode)
  :custom (org-modern-hide-stars " ")
  )


```
I also had to return
